Resources/public/scss/_box.scss

```diff
@@ -93,7 +93,7 @@
 }
 
 .info-box-content {
-    background: $lighter-gray;
+    //background: $lighter-gray;
```

Comment:
@vanhoog Why not remove this line, it's commented right now.

Reply:
@ceesvanegmond line is now removed